scenarios/14e_Saorduc_Swamps.cfg

```diff
@@ -29,7 +29,13 @@
         carryover_percentage=100
         linger_mode=no
     [/endlevel]
-    {END_IF_WITHOUT_ELSE}
+    # TODO: maybe give the option to let the saurians join you?
+    {ELSE}
+    # TODO: test display of these messages:
+    {MESSAGE ({LIZ}) () ({LIZ}) _"We are more than jusssst training fodder for your army!"}
+    {MSGW_Gawen _"I'm sorry, but that's just the way things go."}
```

Comment:
This one doesn't sound like Gawen either tbh.

Reply:
Yeah I didn't really know what to have him say here...

Reply:
> I'm sorry, but that's the way it is.

Reply:
changed in d8d2616